2016-11-28 14:45 DEBUG    manager                       Figuring out config load paths
2016-11-28 14:45 DEBUG    manager                       Found config: /Users/eivindml/.flexget/config.yml
2016-11-28 14:45 DEBUG    manager                       Config file /Users/eivindml/.flexget/config.yml selected
2016-11-28 14:46 CRITICAL task          ezrss.it        BUG: Unhandled error in plugin make_html: [Errno 2] No such file or directory: u'/Users/eivindml/public_html/series.html'
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/FlexGet-1.2.495-py2.7.egg/flexget/task.py", line 438, in __run_plugin
    return method(*args, **kwargs)
  File "/Library/Python/2.7/site-packages/FlexGet-1.2.495-py2.7.egg/flexget/event.py", line 22, in __call__
    return self.func(*args, **kwargs)
  File "/Library/Python/2.7/site-packages/FlexGet-1.2.495-py2.7.egg/flexget/plugins/output/html.py", line 41, in on_task_output
    with open(output, 'w') as f:
IOError: [Errno 2] No such file or directory: u'/Users/eivindml/public_html/series.html'
